base-server.js

```diff
@@ -281,6 +282,37 @@ BaseServer.prototype = {
   },
 
   /**
+   * Load options from both CLI and environment
+   *
+   * @param {object} process Current process.
+   * @param {object} defaults Default options.
+   * @return {object} options.
+   *
+   * @example
+   * options = app.loadOptions(process, defaults)
+   *
+   */
+  loadOptions: function loadOptions (process, defaults) {
+    program
+      .option('-p, --port [port]', 'Set up a port')
```

Comment:
You don’t really use argument description. try to add some info like “port to listen”.